controllers/application/deployment.go

```diff
@@ -13,6 +13,8 @@ import (
 	ctrlutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
+// Adding an argocd external link constant
+const ( AnnotationKeyLinkPrefix = "link.argocd.argoproj.io/external-link" )
```

Comment:
```suggestion
const (
	AnnotationKeyLinkPrefix = "link.argocd.argoproj.io/external-link"
)
```

or

```suggestion
const AnnotationKeyLinkPrefix = "link.argocd.argoproj.io/external-link"
```

See also `go fmt` which automatically formats code according to the [accepted format](https://go.dev/doc/effective_go#formatting).